labelbox/data/annotation_types/data/tiled_image.py

```diff
@@ -46,16 +46,14 @@ class EPSG(Enum):
 
 
 class TiledBounds(BaseModel):
-    """ Bounds for a tiled image asset related to the relevant epsg. 
+    """ Bounds for a tiled image asset related to the relevant epsg.
 
-    Bounds should be Point objects. Currently, we support bounds in EPSG 4326.
-
-    If version of asset is 2, these should be [[lat,lng],[lat,lng]]
-    If version of asset is 1, these should be [[lng,lat]],[lng,lat]]
+    Bounds should be Point objects.
+    Currently, we support bounds in EPSG 4326. These should be [[lat,lng],[lat,lng]]
```

Comment:
Scrap this because technically any bounds that correspond to an epsg that we support is valid